.github/workflows/greetings.yml

```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
```

Comment:
Why ubuntu-latest?

Reply:
I went with the default: ubuntu-latest as this is not running the code, just triggering the actions. Once we start working with code actions, then we can test this out across different OSs.